tests/test_lora.py

```diff
@@ -421,7 +419,7 @@ def test_lora_merge_with_quantize():
     assert torch.equal(attn_proj.linear.weight, weight_after)
 
     # check that `W_after = W_initial + (A x B)`
-    delta_w = (attn_proj.lora_B @ attn_proj.lora_A) * attn_proj.scaling
+    delta_w = attn_proj.get_lora_AB()
     # dequantize initial weight and sum with delta_w
     initial_weight_data = (
```

Comment:
Despite Carlos's [suggestion](https://github.com/Lightning-AI/lit-gpt/pull/771#discussion_r1425619223) I decided to not create such a function, as of right now the logic that is specific to quantized weights is hidden in merge method. If in addition to all existing methods there be will another one, very specific to BNB's logic, that might be confusing. At least in my opinion.

Another reason to not do it is so in tests I can update quantized weight in not the most efficient, but the most robust way: move to CPU, call .to method and thus quantized weights, in contrast to `merge` method where I call .cuda method directly.
It helps to find issues if that direct call doesn't work for some reason.
As an example, my initial idea was to simply reassign the `self.linear.weight.data` and call `.cuda`, but that didn't work. I assume that's because [this](https://github.com/TimDettmers/bitsandbytes/blob/a06a0f6a08cb23754b110359a109e069fa97ce9e/bitsandbytes/nn/modules.py#L147) isn't executed when I simply assign weight without creating a new Params4bit class.
And this test helped to find this issue.